packages/manager/modules/dbaas-logs/src/logs/empty/index.less

```diff
@@ -0,0 +1,31 @@
+.ldp-project-empty {
```

Comment:
**FYI**: we tend to avoid using custom style.

**thought:**  Having custom style mean probably that you miss some functionalities from [`@ovh-ux/ui-kit`](https://ovh.github.io/ovh-ui-kit/) package.
If it's the case, please don't hesitate to raise an Issue.

It can be considered on a separate Pull Request (non-blocking).

Reply:
this empty state can be replaced soon with another PR with a standalone empty state. (https://github.com/ovh/manager/pull/3846)